src/cryptoadvance/specter/specter.py

```diff
@@ -156,6 +157,8 @@ def cleanup_on_exit(self, signum=0, frame=0):
             if not node.external_node:
                 node.stop()
 
+        self.service_manager.execute_ext_callbacks(callbacks.flash, signum, frame)
```

Comment:
should be `callbacks.cleanup_on_exit` i guess. Also please don't assume the service_manager is available on the specter-object. This is a bit tricky as specter is not "ServiceManagerAware". E.g. if someone is using specter in some kind of cli-project in order to generate an address, there is not even flask available.
So please make it failsafe.

Reply:
Thanks for noticing that.  Done in  e8ec3877

Reply:
In  514401b1  I made it that specter stays unaware of the service_manager.